radis/lbl/broadening.py

```diff
@@ -1567,9 +1990,8 @@ def _init_w_axis(w_dat, log_p):
                 line_profile_LDM[l] = {}
                 for m in range(len(wL)):
                     wV_ij = olivero_1977(wG[l], wL[m])  # FWHM
-                    lineshape = voigt_lineshape(
-                        wbroad_centered, wL[m] / 2, wV_ij / 2, jit=jit
-                    )  # FWHM > HWHM
+                    #lineshape = voigt_lineshape(wbroad_centered, wL[m] / 2, wV_ij / 2, jit=jit)  # FWHM > HWHM
+                    lineshape = voigt_lineshape_xiFactor(wbroad_centered, wL[m] / 2, wV_ij / 2, xi_factor, jit=jit)  # with xi_factor
                     line_profile_LDM[l][m] = lineshape
```

Comment:
@pgokulak It might be possible to avoid rewriting existing code such as voigt_lineshape
For instance : 

```
lineshape = voigt_lineshape(...) # keep the default
lineshape *= self.add_chi_factor(self.input.molecule)  
```

Where most of the new code is added in  `add_chi_factor`:  

```
def add_chi_factor(molecule, w_centered, wv):
     """molecule, wavenumber array, voigt width"""
    if molecule == 'CO2':
        pressure_mbar = self.input.pressure_mbar
        mole_fraction = self.input.mole_fraction  # note, might be a dictionary if mulitple molecules ? check later
        Tgas = self.input.Tgas
        Tref = self.input.Tref

        xi_factor = get_xiFactor(pressure_mbar*1e-3, mole_fraction, Tgas, Tref)

     # xi_factor correction for pseudo-Lorentzian of Westlye et al.(2022)
        gamma_v = wv/2 # Voigt half-width
        alpha_xi = xi_factor*np.sin(np.pi/xi_factor)/(2*np.pi*gamma_v) # see see Westlye et al. (2022); Eqn(6)
        lineshape_correction = alpha_xi / (1 + abs(w_centered/gamma_v)**xi_factor) # see see Westlye et al. (2022); Eqn(5)

        return lineshape_correction
    else:
        raise NotImplementedError
```

This would make it easier to maintain the codebase